tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -913,3 +917,74 @@ func createStaticPvAndPvcInParallel(client clientset.Interface, ctx context.Cont
 
 	}
 }
+
+// triggerFullSyncInParallel triggers full sync on demand in parallel
+func triggerFullSyncInParallel(ctx context.Context, client clientset.Interface, wg *sync.WaitGroup) {
+	defer wg.Done()
+	restConfig := getRestConfigClient()
+	crd := getTriggerFullSyncCrd(ctx, client, restConfig)
+	updateTriggerFullSyncCrd(ctx, restConfig, crd)
+	err := waitForFullSyncToFinish(client, ctx, restConfig)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred(), "Full sync did not finish in given time")
+	crd = getTriggerFullSyncCrd(ctx, client, restConfig)
+	updateTriggerFullSyncCrd(ctx, restConfig, crd)
+}
+
+// getTriggerFullSyncCrd fetches full sync crd from the list of crds in k8s cluster
+func getTriggerFullSyncCrd(ctx context.Context, client clientset.Interface,
+	restConfig *rest.Config) triggercsifullsyncv1alpha1.TriggerCsiFullSync {
+	cnsOperatorClient, err := k8s.NewClientForGroup(ctx, restConfig, cnsoperatorv1alpha1.GroupName)
+	gomega.Expect(err).NotTo(gomega.HaveOccurred())
+	crds := &triggercsifullsyncv1alpha1.TriggerCsiFullSyncList{}
+	err = cnsOperatorClient.List(ctx, crds)
```

Comment:
Why do we need to list the items here, we know that the crd has a constant name, we can use a `Get()` here an avoid the following loop as well.

Reply:
Done